### PR TITLE
fix savefig extensions

### PR DESCRIPTION
**Pull request by daschw**

### CODE REVIEW

File: src/output.jl [hunk 1]
```diff
@@ -88,10 +88,13 @@ const _savemap = Dict(
     "tex" => tex,
     "json" => json,
     "html" => html,
+    "tex" => tex,
```
Comment (BeastyBlacksmith) [inline src/output.jl:91]: thats the same as line 88

**Comment by BeastyBlacksmith** [comment]
Out of curiousity. What was the issue?

File: src/output.jl (hunk 1 above)
Comment (daschw) [reply src/output.jl:91]: oh, yeah

**Comment by daschw** [comment]
apparantly all files are saved as `fn.ext.ext` instead of `fn.ext`

**Comment by daschw** [comment]
... on julia 1.0

**Pull request merged**